doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -43,6 +43,7 @@ nRF9160
     * Added more LwM2M objects.
     * LwM2M sensor objects now uses the actual sensors available to the Thingy:91. If the nRF9160 DK is used, it uses simulated sensors instead.
     * Added possibility to poll sensors and notify the server if the measured changes are large enough.
+    * Added possibility to use full modem firmware update.
```

Comment:
```suggestion
    * Added support for polling sensors and notifying the server if the measured changes are large enough.
    * Added support for full modem firmware update.
```

Reply:
The upper line wasn't part of my PR, but changed that also to the suggested format.